packages/tizen_log/lib/tizen_log.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+import 'package:ffi/ffi.dart';
+import 'package:flutter/foundation.dart';
+
+typedef _DLogNative = Void Function(
+    Int32, Pointer<Utf8>, Pointer<Utf8>, Pointer<Utf8>);
+typedef _DLog = void Function(int, Pointer<Utf8>, Pointer<Utf8>, Pointer<Utf8>);
+
+/// Provides the ability to use Tizen dlog logging service.
+/// More info: https://docs.tizen.org/application/native/guides/error/system-logs/
+class Log {
+  Log._();
+
+  static final DynamicLibrary _library = DynamicLibrary.open('liblog.so');
+  static final _DLog _dlogPrint =
+      _library.lookup<NativeFunction<_DLogNative>>('dlog_print').asFunction();
+  static final RegExp _stackTraceRegExp =
+      RegExp(r'^#(\d+)\s+(.+)\((.+):(\d+):(\d+)\)$', multiLine: true);
+
+  /// For displaying a file name, function name and line number in the logs,
+  /// use --dart-define=DEBUG_MODE=debug flag in build time.
+  /// flutter-tizen run --dart-define=DEBUG_MODE=debug
+  static const String debugMode =
+      String.fromEnvironment('DEBUG_MODE', defaultValue: 'no');
```

Comment:
Why is the expected value "debug", but not something like "true" or "enabled"?

Also you could make this property [private](https://dart.dev/guides/language/effective-dart/design#prefer-making-declarations-private) or even merge with `isDebugEnabled`.

Reply:
Could you answer this question? Will there be any other mode than "debug"?

Reply:
Reminding again. @m-bialek

Reply:
As far as I can see, I merged this property with `isDebugEnabled` as you suggested but I forgot to reply to this comment. I am sorry for that.

Reply:
I copied this solution from another project. I do not think there will be other modes than debug.

Reply:
I changed String to bool. Now the user can write `--dart-define=DEBUG_MODE=true` to enable debug mode.